Mac build problem

Figure out what's going on here:

Downloaded: https://repo.maven.apache.org/maven2/org/eclipse/tycho/org.eclipse.osgi/3.10.101.v20150820-1432/org.eclipse.osgi-3.10.101.v20150820-1432.jar (1276 KB at 2323.7 KB/sec)
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for com.google.gcp.eclipse:com.google.gcp.eclipse.appengine.localserver:94.0.0-SNAPSHOT: Could not find artifact com.google.gcp.eclipse:trunk:pom:0.0.1-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 5, column 11
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project com.google.gcp.eclipse:com.google.gcp.eclipse.appengine.localserver:94.0.0-SNAPSHOT (/Users/elharo/gcloud-eclipse-tools/plugins/com.google.gcp.eclipse.appengine.localserver/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for com.google.gcp.eclipse:com.google.gcp.eclipse.appengine.localserver:94.0.0-SNAPSHOT: Could not find artifact com.google.gcp.eclipse:trunk:pom:0.0.1-SNAPSHOT and 'parent.relativePath' points at wrong local POM @ line 5, column 11 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException

--- a/plugins/com.google.gcp.eclipse.appengine.localserver/pom.xml
+++ b/plugins/com.google.gcp.eclipse.appengine.localserver/pom.xml
@@ -5,7 +5,7 @@
   <parent>
     <groupId>com.google.gcp.eclipse</groupId>
     <artifactId>trunk</artifactId>
-    <version>0.0.1-SNAPSHOT</version>
+    <version>94.0.0-SNAPSHOT</version>
     <relativePath>../../</relativePath>
   </parent>
   <groupId>com.google.gcp.eclipse</groupId>

